core/src/banking_stage/transaction_scheduler/in_flight_tracker.rs

```diff
@@ -34,6 +34,7 @@ impl InFlightTracker {
     }
 
     /// Returns the number of cus that are in flight for each thread.
+    #[allow(dead_code)]
```

Comment:
Should we just move this to be within test module ?

Reply:
This is something I still plan to use for load-balancing in the future. The main reason its' unused now is because of innaccurate requests. Recent metrics showing these are becoming less innaccurate (or scheduler is prioritizing those that are more accurate).